### PR TITLE
CRAN updates

### LINKED ISSUE #56: New errors with EpiModel v2.3.0
There are new errors arising from the publication of EpiModel v2.3.0 on CRAN. 
https://cran.r-project.org/web/checks/check_results_tergmLite.html

@chad-klumb : could you check these and update the unit tests as necessary. At a minimum, we can use some conditional logic to only test if EpiModel v is < 2.3.0. 

I'd like to keep tergmLite up on CRAN for the time being in case any users are still using older versions of EpiModel.

### PR DESCRIPTION
**Pull request by chad-klumb**
skip examples and check EpiModel version on tests

should fix CRAN issues

closes #56

### CODE REVIEW

File: tests/testthat/test-networkLite.R [hunk 1]
```diff
@@ -1,208 +1,220 @@
...
-  nw_mple_ergm <- ergm(nw ~ edges + nodefactor("a"),
-                       target.stats = as.integer(target_stats[-length(target_stats)]),
-                       constraints = di_constraints)
-  set.seed(0)
-  nwL_mple_ergm <- ergm(nwL ~ edges + nodefactor("a"),
-                        target.stats = as.integer(target_stats[-length(target_stats)]),
-                        constraints = di_constraints)
-  expect_equal(coef(nw_mple_ergm), coef(nwL_mple_ergm))
+  library("EpiModel")
+  if (packageVersion("EpiModel") >= "2.1.0" & packageVersion("EpiModel") < "2.2.0") {
```
Comment (smjenness) [inline tests/testthat/test-networkLite.R:3]: Shouldn't this be < 2.3.0? EpiModel 2.2.0 was the prior version on CRAN, and tergmLite was working ok with that.

**Comment by chad-klumb** [comment]
I used the same version numbers as in the other tests, but I will change it to 2.3.0.

**Comment by chad-klumb** [comment]
it looks like 2.2.0 is when tergmLite functionality was merged into EpiModel, so logically requiring version < 2.2.0 would make more sense to me, even if the tests appear to pass with some later version(s)

**Comment by smjenness** [comment]
Ok, that's fine. You can revert the last commit.

**Comment by chad-klumb** [comment]
done

**Comment by smjenness** [comment]
Thanks @chad-klumb !

**Comment by chad-klumb** [comment]
you're welcome

final note that compatibility with EpiModel version < 2.2.0 is based on CRAN version numbers (not all intermediate github commits)

**Pull request merged**